harness/tests/determined/common/experimental/test_trial.py

```diff
@@ -0,0 +1,40 @@
+from typing import Callable
+
+import pytest
+import responses
+
+from determined.common import api
+from determined.common.experimental import trial
+from tests.fixtures import api_responses
+
+_MASTER = "http://localhost:8080"
+
+
+@pytest.fixture
+def standard_session() -> api.Session:
+    return api.Session(master=_MASTER, user=None, auth=None, cert=None)
+
+
+@pytest.fixture
+def make_trialref(standard_session: api.Session) -> Callable[[int], trial.Trial]:
+    def _make_trialref(trial_id: int) -> trial.Trial:
+        return trial.Trial(trial_id=trial_id, session=standard_session)
+
+    return _make_trialref
+
+
+@responses.activate
+def test_trial_includes_summary_metrics(make_trialref: Callable[[int], trial.Trial]) -> None:
```

Comment:
Does every well-formed `v1GetTrialResponse` have an `avg_metrics` and a `validation_metrics`? [The python](https://github.com/determined-ai/determined/blob/main/harness/determined/common/api/bindings.py#L536) only enforces the existence of summaryMetrics, best I can tell. Without that enforcement, as implemented this is also a hidden test of what's in `trial.json`.

Reply:
Playing around with trials a little, it looks like those there is no such enforcement.